src/containers/Footer/index.jsx

```diff
@@ -24,31 +24,6 @@ const Footer = (props) => {
           <a href="https://xrpl.org/impact.html" className="footer-link">
             Impact
           </a>
-          <a
-            href="https://xrpl.org/carbon-calculator.html"
-            className="footer-link"
-          >
-            Carbon Calculator
-          </a>
-        </div>
-
-        <div className="footer-link-section">
-          <div className="footer-section-header">Explore</div>
-          <a href="https://xrpl.org/index.html" className="footer-link">
-            News
-          </a>
-          <a href="https://xrpl.org/wallet.html" className="footer-link">
-            Wallets
-          </a>
-          <a href="https://xrpl.org/exchanges.html" className="footer-link">
-            Exchanges
-          </a>
-          <a href="https://xrpl.org/businesses.html" className="footer-link">
-            XRPL Businesses
-          </a>
-          <a href="https://livenet.xrpl.org/" className="footer-link">
```

Comment:
I think it's OK to leave this link here (or something similar) as it's just a convenient way to go back to the explorer homepage (when you've scrolled to the bottom of a page)

Reply:
Should it go under build?

Reply:
I think it's fine anywhere - whatever you like :)